### DISCUSSION TITLE
RTMP player alternative to ExoPlayer?

### DISCUSSION THREAD

**Discussion by AlexUrrutia** [pedroSG94/RootEncoder#1195] [General]
Hello @pedroSG94 , do you know of any RTMP player for **Android** other than **ExoPlayer**? I want to play multiple **RTMP** live streams in **Android** but **ExoPlayer** is too heavy and doesn't work well using **multiple instances**.

_I just need to play RTMP live, no need extra libraries like RTSP etc._

Thanks in advance!

**Comment by pedroSG94** [comment]
Hello,

I only know 3 RTMP players for Android:
- ExoPlayer
- IJKPlayer(based on ffmpeg): https://github.com/bilibili/ijkplayer
- VLC: https://code.videolan.org/videolan/vlc-android
